src/application-base/ApplicationBase.module.scss

```diff
@@ -1,15 +1,9 @@
 :local {
-  .application-theme-provider {
+  .application-base {
     position: relative;
-  }  
-    
-  .application-theme-provider.fill {
-    height: 100%;
-  } 
+  }
 
   .application-base.fill {
     height: 100%;
-    overflow: auto;
-    -webkit-overflow-scrolling: touch;
```

Comment:
do we still need these styles?

Reply:
Funny thing, we weren't applying them before. So I checked with @tbiethman, No we don't need them.